README.markdown

```diff
@@ -11,21 +11,20 @@ Installation:
 
 Usage:
 
-	upstream backend {                                                                              
-		jdomain www.baidu.com; #port=80                                                             
-		#jdomain www.baidu.com port=8080; #port=8080
+	upstream backend {
+		refreshed_hostname www.baidu.com;
 	}
-	
+
 	resolver 8.8.8.8; #Your Local DNS Server
-                                                                                                    
-Jdomain: 
 
-	* Syntax: jdomain <domain-name> [port=80] [max_ips=20] [interval=1] [retry_off]
-	* Context:    upstream                                                                          
-	* port:       Backend's listening port.                                                         
+Hostname:
```

Comment:
nit: this should probably be `<directive>` (as a literal)

The README as a whole could likely be organized a bit better but this seems to be the directive docs so it should match the directive exactly at likely as a markdown heading rather than having the `:` at the end.